data/avro-schema/src/main/resources/avro/net/corda/data/ledger/utxo/token/selection/data/TokenClaim.avsc

```diff
@@ -9,6 +9,12 @@
         "type": "string",
         "doc": "Unique identifier for the claim"
       },
+      {
+        "name": "claimTimestamp",
+        "type": ["null", "long"],
+        "default": null,
```

Comment:
What does it mean if a claim does not have a timestamp? Will a claim with no timestamp ever expire?

Reply:
in the runtime-os change that uses this, if it finds a old 5.0 version state it defaults the timestamp to the current time.